.github/workflows/cicd.yml

```
@@ -2,6 +2,11 @@ name: Continuous Integration & Delivery
 
 on:
   push:
+    branches: [develop, main, bugfix/*, feature/*]
+    paths-ignore: ["docs/**", "examples/**"]
+  pull_request:
+    branches: [develop, main]
+    paths-ignore: ["docs/**", "examples/**"]
     branches: [ develop, main, bugfix/*, feature/* ]
```

actionlint (1.7.8)

key "branches" is duplicated in "pull_request" section. previously defined at line:8,col:5

(syntax-check)